brain-marks/ContentView.swift

```diff
@@ -8,10 +8,17 @@
 import SwiftUI
 
 struct ContentView: View {
+    let migrationService: MigrationService = MigrationService()
+
     @State private var showAddSheet = false
 
     var body: some View {
         CategoryList()
+            .onAppear {
+                if migrationService.checkIfMigrationShouldRun() {
```

Comment:
Let's add a telemetry signal that a user's data was migrated

`migrationFromAmplifyToCDPerformed`

Reply:
Done ✅